src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,481 @@
+-------------------------------------------------------------
+-- Drop each of the schemas used by SDT and all their contents
+--------------------------------------------------------------
+--DROP SCHEMA IF EXISTS "SDT_USER" CASCADE;
+--DROP SCHEMA IF EXISTS "SDT_BATCH_USER" CASCADE;
+--DROP SCHEMA IF EXISTS "SDT_OWNER" CASCADE;
+
+-------------------------------------------------------------
+-- Drop each of the users used by SD
+--------------------------------------------------------------
+---- non schema owners first
+--DROP USER IF EXISTS "SDT_USER";
+--DROP USER IF EXISTS "SDT_BATCH_USER";
+---- now schema owners
+--DROP USER IF EXISTS "SDT_OWNER";
+
+-------------------------------------------------------------
+-- Create the schemas used by SDT and all their contents
+--------------------------------------------------------------
+CREATE SCHEMA "SDT_OWNER";
+CREATE SCHEMA "SDT_USER";
+CREATE SCHEMA "SDT_BATCH_USER";
+
+------------------------------------------------
+-- Create each of the schemas used by SDT
+------------------------------------------------
+CREATE USER "SDT_OWNER";
+GRANT POSTGRES TO "SDT_OWNER";
+
+CREATE USER "SDT_USER";
+GRANT POSTGRES TO "SDT_USER";
+CREATE USER "SDT_BATCH_USER";
+GRANT POSTGRES TO "SDT_BATCH_USER";
+
+------------------------------------------------
+-- Create tables for SDT_OWNER
+------------------------------------------------
+CREATE TABLE "SDT_OWNER"."BULK_CUSTOMERS"
+   ("BULK_CUSTOMER_ID" NUMERIC NOT NULL, "SDT_CUSTOMER_ID" NUMERIC(8),
+	"VERSION_NUMBER" NUMERIC DEFAULT 0 ) ;
+
+CREATE TABLE "SDT_OWNER"."BULK_CUSTOMER_APPLICATIONS"
+   ("BULK_CUSTOMER_APPLICATIONS_ID" NUMERIC NOT NULL,
+	"BULK_CUSTOMER_ID" NUMERIC,	"TARGET_APPLICATION_ID" NUMERIC,
```

Comment:
Replace the tab after the first comma with a space.  There's quite a few more instances of this throughout the file.

Reply:
Not necessary but done.

Reply:
It is a cosmetic change, but it does ensure that a consistent spacing scheme is used throughout the file.

There are still seven tabs left:
Line 42 - after comma at the end of '"BULK_CUSTOMER_ID" NUMERIC,'
Line 43 - after comma at the end of '"CUSTOMER_APPLICATION_ID" VARCHAR(32),'
Line 46 - after first two commas (they look like spaces but are tabs)
Line 49 - after comma at the end of '"COMPLETED_DATE" TIMESTAMP(6),'
Line 50 - after comma at the end of 'VERSION_NUMBER" NUMERIC DEFAULT 0,' (looks like a space but is a tab)
Line 80 - after comma at end of '"SERVICE_TYPE_ID" NUMERIC,' (looks like a space but is a tab)

Reply:
The CREATE table statements have been changed to have each column on a new line so these comments no longer apply.